com.yubico.YubiOath.json

```diff
@@ -0,0 +1,257 @@
+{
+    "app-id": "com.yubico.YubiOath",
+    "runtime": "org.kde.Platform",
+    "sdk": "org.kde.Sdk",
+    "runtime-version": "5.12",
+    "command": "yubioath-desktop",
+    "cleanup": [
+	"/include",
+	"/lib/pkgconfig"
+    ],
+    "finish-args": [
+        "--socket=x11",
+	"--socket=pcsc",
+	/* We need to be able to list usb devices, even when using CCID */
+	"--device=all"
+    ],
+    "modules": [
+	{
+	    "name": "libusb",
+	    "config-opts": ["--enable-udev=no"],
+	    "cleanup": [
+		"/lib/libusb-1.0.so"
+	    ],
+	    "sources": [
+		{
+                    "type": "archive",
+		    "url": "https://github.com/libusb/libusb/releases/download/v1.0.22/libusb-1.0.22.tar.bz2",
+		    "sha256": "75aeb9d59a4fdb800d329a545c2e6799f732362193b465ea198f2aa275518157"
+		}
+	    ]
+	},
+	{
+	    "name": "pcsc-lite",
+	    "config-opts": [
+		"--disable-libudev",
+		"--disable-libsystemd",
+		"--without-systemdsystemunitdir"],
+	    "post-install": [
+		"rm /app/sbin/pcscd",
+		"rmdir /app/sbin || true"
+	    ],
+	    "cleanup": [
+		"/lib/libpcscspy.so",
+		"/lib/libpcsclite.so"
+	    ],
+	    "sources": [
+		{
+                    "type": "archive",
+                    "url": "https://github.com/LudovicRousseau/PCSC/archive/pcsc-1.8.24.tar.gz",
```

Comment:
latest is [1.8.25](https://github.com/LudovicRousseau/PCSC/releases/tag/pcsc-1.8.25)

Reply:
Fixed